ang/crmMosaico/BlockPreview.html

```diff
@@ -1,60 +1,53 @@
 <!--
 Vars: mailing:obj, testContact:obj, testGroup:obj, crmMailing:FormController
 -->
-<div class="panel panel-default crm-mosaico-test">
-  <div class="panel-heading"><h3 class="panel-title">{{ts('Preview/ Test')}}</h3></div>
-  <div class="panel-body">
+<div class="form-group">
+  <label>{{ts('Preview:')}}</label>
 
-    <div class="form-group">
-      <label>{{ts('Preview:')}}</label>
-
-      <div ng-show="!mailing.body_html && !mailing.body_text">
-        <em>({{ts('No content to preview')}})</em>
-      </div>
-      <div ng-hide="!mailing.body_html" class="btn-group btn-group-justified">
-        <div class="btn-group">
-          <button class="btn btn-primary" ng-disabled="crmMailing.$invalid" ng-click="doPreview('html')">{{ts('Preview as HTML')}}</button>
-        </div>
-      </div>
-      <div ng-hide="!mailing.body_html && !mailing.body_text" style="margin-top: 1em;" class="btn-group btn-group-justified">
-        <div class="btn-group">
-          <!--<a class="btn btn-sm btn-primary"  ng-disabled="crmMailing.$invalid" ng-click="doPreview('text')">{{ts('Preview as Plain Text')}}</a>  -->
-          <button class="btn btn-primary" ng-disabled="crmMailing.$invalid" ng-click="doPreview('text')">{{ts('Preview as Plain Text')}}</button>
-        </div>
-      </div>
+  <div ng-show="!mailing.body_html && !mailing.body_text">
+    <em>({{ts('No content to preview')}})</em>
+  </div>
+  <div ng-hide="!mailing.body_html" class="btn-group btn-group-justified">
+    <div class="btn-group">
+      <button class="btn btn-primary" ng-disabled="crmMailing.$invalid" ng-click="doPreview('html')">{{ts('Preview as HTML')}}</button>
     </div>
-
-    <div class="form-group">
-      <label for="preview_test_email">{{ts('Send test email:')}} <a crm-ui-help="hs({id: 'test', title: ts('Test Email')})"></a> </label>
-
-      <input
-      name="preview_test_email"
-      type="email"
-      class="form-control margin-bottom-10"
-      ng-model="testContact.email"
-      placeholder="example@example.org" />
-
-      <!--fa-paper-plane-->
-      <button class="btn btn-sm btn-primary" title="{{crmMailing.$invalid || !testContact.email ? ts('Complete all required fields first') : ts('Send test message to %1', {1: testContact.email})}}" ng-disabled="crmMailing.$invalid || !testContact.email" ng-click="doSend({email: testContact.email})">{{ts('Send test')}}</button>
+  </div>
+  <div ng-hide="!mailing.body_html && !mailing.body_text" style="margin-top: 1em;" class="btn-group btn-group-justified">
+    <div class="btn-group">
+      <!--<a class="btn btn-sm btn-primary"  ng-disabled="crmMailing.$invalid" ng-click="doPreview('text')">{{ts('Preview as Plain Text')}}</a>  -->
```

Comment:
Actually i didn't change this. i only removed the first 3 lines 

![screen shot 2017-04-04 at 5 52 44 pm](https://cloud.githubusercontent.com/assets/2423218/24666126/b71f46ea-195f-11e7-9114-411721284e94.png)

But i'll update this in my next commit